protocol/src/main/java/org/geysermc/mcprotocollib/auth/GameProfile.java

```diff
@@ -390,7 +389,7 @@ public static class Texture {
          */
         public Texture(String url, Map<String, String> metadata) {
             this.url = url;
-            this.metadata = new HashMap<>(metadata);
+            this.metadata = metadata;
```

Comment:
Why was this changed? Is the metadata supposed to be mutable now? (Assuming that was the purpose of the copy)

Reply:
IntelliJ would claim metadata is never null while it can be null due to gson. Idk how gson does it, but it's able to set the metadata field to null.

Reply:
Interesting - is null metadata valid?

Reply:
yeah when a skin is wide, metadata field is missing from textures, if it's slim the metadata field is present

Reply:
<https://github.com/tinytengu/minecraft-authlib/blob/main/src/main/java/com/mojang/authlib/minecraft/MinecraftProfileTexture.java> Got curious and checked, seems to be what minecraft's authlib is doing, so that looks fine.